util/settings/settings.go

```diff
@@ -81,6 +81,7 @@ type RepoCredentials struct {
 	PasswordSecret        *apiv1.SecretKeySelector `json:"passwordSecret,omitempty"`
 	SSHPrivateKeySecret   *apiv1.SecretKeySelector `json:"sshPrivateKeySecret,omitempty"`
 	InsecureIgnoreHostKey bool                     `json:"insecureIgnoreHostKey,omitempty"`
```

Comment:
should this be annotated with `// DEPRECATED`?

Reply:
I tried using it in the comment, but the linter complained heavily about it.